Python/Product/Common/Common.csproj

```diff
@@ -63,6 +64,7 @@
     <Compile Include="Infrastructure\StringListReader.cs" />
     <Compile Include="Infrastructure\TaskExtensions.cs" />
     <Compile Include="Infrastructure\TestEnvironment.cs" />
+    <Compile Include="Infrastructure\VSWin32Window.cs" />
```

Comment:
I know this isn't actually VS specific, so can we rename it to not look like it is? Or we should have a VS-specific common DLL somewhere it can go in.